{{cookiecutter.project_slug}}/home/templates/users/user_list.html

```diff
@@ -0,0 +1,17 @@
+{% raw %}{% extends "base.html" %}
+{% load static i18n %}
+{% block title %}Members{% endblock %}
+
+{% block content %}
+<div class="container">
+  <h2>Users</h2>
+
+  <div class="list-group">
+    {% for user in user_list %}
```

Comment:
and, this is object_list unless we modify context_object_name to user_list

Reply:
The list template isn't actually used so I'm removing it.